server/src/controllers/Venue/resolver.ts

```diff
@@ -102,6 +104,86 @@ export class VenueResolver {
     @Arg('_onlyUsedForAuth', () => Int) _onlyUsedForAuth: number,
   ): Promise<{ id: number }> {
     // TODO: handle deletion of non-existent venue
+    const users = await prisma.users.findMany({
+      where: {
+        user_events: {
+          some: {
+            event: {
+              canceled: false,
+              ends_at: { gt: new Date() },
+              venue_id: id,
+            },
+            subscribed: true,
+          },
+        },
+      },
+    });
```

Comment:
```suggestion
    const users = await prisma.users.findMany({
      where: {
        user_events: {
          some: {
            event: {
              canceled: false,
              ends_at: { gt: new Date() },
              venue_id: id,
            },
            subscribed: true,
            rsvp: { name: 'yes' },
          },
        },
      },
      include: {
        user_events: {
          where: {
            event: {
              canceled: false,
              ends_at: { gt: new Date() },
              venue_id: id,
            },
            subscribed: true,
            rsvp: { name: 'yes' },
          },
        },
      },
    });
```
:eyes:
- Filter users having at least one `user_events` with:
  - subscribed: true
  - rsvp: yes
  - Event having:
    - canceled: false
    - ends_at: { gt: new Date() }
    - venue_id: id
- For filtered users, include all `user_events` with:
  - subscribed: true
  - rsvp: yes
  - Event having:
    - canceled: false
    - ends_at: { gt: new Date() }
    - venue_id: id

Sorry! :eyes: